tests/all/js_globals/Math.rs

```diff
@@ -293,6 +294,117 @@ fn clz32() {
         .test()
 }
 
+#[test]
+fn cos() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js;
+
+            #[wasm_bindgen]
+            pub fn cos(x: f32) -> js::Number {
+                js::Math::cos(x)
+            }
+        "#)
+        .file("test.ts", r#"
+            import * as assert from "assert";
+            import * as wasm from "./out";
+
+            export function test() {
+                assert.equal(wasm.cos(0), 1);
+                // other assertions failing due to rounding errors
```

Comment:
If want to make more assertions what you could do is the following:
```
assert.equal(wasm.cos(1.34), Math.cos(1.34)))
```
The same goes for the other assertions where you have a very precise float. Doing the above could make it a bit more easy to read as well :)